concent_api/core/tests/test_unit_authentication_utilities.py

```diff
@@ -0,0 +1,214 @@
+from django.test                    import override_settings
+
+from golem_messages                 import message
+from golem_messages.shortcuts       import dump
+from golem_messages.shortcuts       import load
+
+from core.exceptions                import Http400
+from core.validation                import validate_golem_message_client_authorization
+from core.validation                import validate_golem_message_signed_with_key
+from core.validation                import validate_list_of_identical_task_to_compute
+from core.tests.utils               import ConcentIntegrationTestCase
+from utils.shortcuts                import load_without_public_key
+from utils.testing_helpers          import generate_ecc_key_pair
+
```

Comment:
Generally, all the tests above are just simple unit tests (which is great!), so it would be better to make them more lightweight by inheriting from `unittest.TestCase` and not `ConcentIntegrationTestCase`

Reply:
:+1: